tests/integration/helpers.py

```diff
@@ -135,18 +138,23 @@ async def run_mongo_op(
         mongo_cmd = f"mongosh --quiet --eval '{mongo_op}' {mongo_uri}{suffix}"
 
     logger.info("Running mongo command: %r", mongo_cmd)
+
+    create_pod_if_not_exists(
+        ops_test.model_name, HELPER_MONGO_POD_NAME, "mongo", f"mongo:{HELPER_MONGO_VERSION}"
+    )
+
+    while not is_pod_ready(ops_test.model_name, HELPER_MONGO_POD_NAME):
+        logger.info("Waiting for pod to be ready...")
+        time.sleep(5)
```

Comment:
oh epic, thank you for this. I think it will also be necessary to port over this fix to k8s charm. Mind also doing that?

Reply:
you mean VM charm 😄 ? Yes, I am planning to do that